### PR TITLE
Make OcDrop rounded / Improve OcList

### PR DESCRIPTION
**Pull request by lookacat**
## Description
We've added a border-radius for the OcDrop 
to make it rounded.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation added/updated

### CODE REVIEW

File: src/components/atoms/OcDrop/OcDrop.vue [hunk 1]
```diff
@@ -201,8 +206,13 @@ export default {
   }
 }
 
+.rounded {
```
Comment (kulmann) [inline src/components/atoms/OcDrop/OcDrop.vue:209]: Please use component namespacing as we don't have scoped styles at the moment. `oc-drop-rounded`.

Comment (lookacat) [reply src/components/atoms/OcDrop/OcDrop.vue:209]: alright thanks for the hint will try to remember next time ^^

File: src/components/atoms/OcDrop/OcDrop.vue [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <template>
-  <div :id="dropId" ref="drop" class="oc-drop oc-box-shadow-medium" @click="$_ocDrop_close">
-    <div v-if="$slots.default" class="oc-card oc-card-body" :class="paddingClass">
+  <div :id="dropId" ref="drop" class="oc-drop oc-box-shadow-medium rounded" @click="$_ocDrop_close">
```
Comment (kulmann) [inline src/components/atoms/OcDrop/OcDrop.vue:2]:
```suggestion
  <div :id="dropId" ref="drop" class="oc-drop oc-box-shadow-medium oc-drop-rounded" @click="$_ocDrop_close">
```

File: src/components/atoms/OcDrop/OcDrop.vue [hunk 3]
```diff
@@ -201,8 +204,13 @@ export default {
   }
 }
 
+.oc-drop-rounded {
+  border-radius: 5px;
+}
+
 .oc-drop {
   width: 300px;
+  border-radius: 5px;
```
Comment (kulmann) [inline src/components/atoms/OcDrop/OcDrop.vue:213]: Delete this line, as the div already has the `oc-drop-rounded` class.

File: src/components/atoms/OcList/OcList.vue [hunk 4]
```diff
@@ -14,5 +14,35 @@ export default {
...
+  margin: 0;
+  padding: 0;
+
+  &-divider > :nth-child(n + 2) {
+    border-top: 1px solid var(--oc-color-border);
+    margin-top: var(--oc-space-small);
+    padding-top: var(--oc-space-small);
+  }
+}
+.oc-list-raw {
```
Comment (kulmann) [inline src/components/atoms/OcList/OcList.vue:39]: Could you create the class one line above as `&-raw` for commonly used scss syntax?

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]
👍

File: src/styles/theme/oc-text.scss [hunk 5]
```diff
@@ -1,12 +1,5 @@
...
-h1,
-h2,
-h3,
-h4,
-h5,
-h6,
-p {
-  color: var(--oc-color-text-default);
+html * {
+  /*color: var(--oc-color-text-default);*/
```
Comment (kulmann) [inline src/styles/theme/oc-text.scss:2]: Hm... What about keeping the old css for the color, and just adding the `html * {}` for the font? Don't we loose text colors with your current changeset?

Comment (lookacat) [reply src/styles/theme/oc-text.scss:2]: it doesnt seem like anything got broken

Comment (pascalwengerter) [reply src/styles/theme/oc-text.scss:2]: Just checked it out locally, rebased on master and used in it in web with a local link. Headings stay black independently of the picked theme, so we need some sort of awareness of the text color variable here. Let me check which approach works best

Comment (pascalwengerter) [reply src/styles/theme/oc-text.scss:2]: Setting the color on the HTML tag messes with quite some other components, so I'd suggest we keep the coloring as it was before and only add the ` html * { font-family }` stuff :)

Comment (lookacat) [reply src/styles/theme/oc-text.scss:2]: alright will do that

File: src/components/atoms/OcList/OcList.vue [hunk 6]
```diff
@@ -9,11 +14,35 @@ export default {
   name: "OcList",
   status: "ready",
   released: "unreleased",
+  props: {
+    raw: {
```
Comment (pascalwengerter) [inline src/components/atoms/OcList/OcList.vue:18]: Could I ask you to add examples for this component, both for the raw and non-raw version?

Comment (lookacat) [reply src/components/atoms/OcList/OcList.vue:18]: yes sure

Comment (lookacat) [reply src/components/atoms/OcList/OcList.vue:18]: added gerneral docs for oclist, raw="true" doesnt seem to have an effect inside ODS docs (in web it does)

**Comment by pascalwengerter** [comment]
@lookacat @kulmann anything left to merge & alpha-release this one? :)

**Pull request merged**